packages/gp2-components/src/organisms/OutputDetailPageHeader.tsx

```diff
@@ -0,0 +1,118 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+import {
+  Card,
+  Display,
+  SharedResearchMetadata,
+  pixels,
+  UsersList,
+  formatDate,
+  lead,
+} from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import { AssociationList } from '../molecules';
+
+const { rem, mobileScreen } = pixels;
+
+const containerStyles = css({
+  padding: `${rem(32)} ${rem(24)}`,
+});
+
+const headerStyle = css({
+  display: 'flex',
+  marginBottom: rem(12),
+});
+
+const associationStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  rowGap: rem(6),
+  gap: rem(6),
+  margin: `${rem(4)} 0 ${rem(32)}`,
+});
+
+const timestampStyles = css({
+  color: lead.rgb,
+  display: 'flex',
+  flexDirection: 'row',
+  whiteSpace: 'pre',
+  fontSize: rem(14),
+  fontWeight: 400,
+  [`@media (max-width: ${mobileScreen.max}px)`]: {
+    flexDirection: 'column',
+  },
+});
+
+type OutputDetailPageHeaderProps = Pick<
+  gp2Model.OutputResponse,
+  | 'addedDate'
+  | 'authors'
+  | 'created'
+  | 'documentType'
+  | 'lastUpdatedPartial'
+  | 'link'
+  | 'mainEntity'
+  | 'projects'
+  | 'subtype'
+  | 'title'
+  | 'type'
+  | 'workingGroups'
+>;
+
+const OutputDetailPageHeader = ({
+  addedDate,
+  authors,
+  created,
+  documentType,
+  lastUpdatedPartial,
+  link,
+  mainEntity,
+  projects,
+  subtype,
+  title,
+  type,
+  workingGroups,
+}: OutputDetailPageHeaderProps) => (
+  <Card padding={false}>
+    <div css={containerStyles}>
+      <SharedResearchMetadata
```

Comment:
I know this was done before, but I noticed that the pills are under <small> element, and the text seems to be smaller than the designs

FIGMA
<img width="550" alt="Screenshot 2023-10-11 at 10 32 57" src="https://github.com/yldio/asap-hub/assets/37268411/d013c344-dab2-494a-9bed-7b0beaae4ea1">


WITH SMALL (font-size 80%)
![Screenshot 2023-10-11 at 10 32 22](https://github.com/yldio/asap-hub/assets/37268411/cdc07b57-e194-4ac1-84ab-a5641ebc5c33)

WITHOUT SMALL (font-size 100%)
![Screenshot 2023-10-11 at 10 33 38](https://github.com/yldio/asap-hub/assets/37268411/397e81f4-6f1f-4418-9b90-b97fd245b17c)